bindings/java/src/org/sleuthkit/datamodel/AbstractFile.java

```diff
@@ -97,6 +97,9 @@ public abstract class AbstractFile extends AbstractContent {
 	protected String sha1Hash;
 	private boolean sha1HashDirty = false;
 	
+	private TskData.CollectedStatus collected; // Collected status of file data	
```

Comment:
should we set this to volatile? we could have one thread setting up the collected status and another invoking save?